R/RLearner_fcregr_autoarima.R

```diff
@@ -0,0 +1,95 @@
+makeRLearner.fcregr.auto.arima = function() {
+  makeRLearnerForecastRegr(
+    cl = "fcregr.auto.arima",
+    package = "forecast",
+    par.set = makeParamSet(
+      makeIntegerLearnerParam(id = "d", lower = 0, upper = Inf),
+      makeIntegerLearnerParam(id = "D", lower = 0, upper = Inf),
+      makeIntegerLearnerParam(id = "max.p", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.q", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.P", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.Q", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.order", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.d", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.D", lower = 0, upper = Inf, default = 1),
+      makeIntegerLearnerParam(id = "start.p", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "start.q", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "start.P", lower = 0, upper = Inf, default = 1),
+      makeIntegerLearnerParam(id = "start.Q", lower = 0, upper = Inf, default = 1),
```

Comment:
setting `tunable = FALSE` would stop people from tuning the autoregressive parameters though right? What if instead we include a note saying,
 "please set the bounds for the autoregressive parameters to be less  than the length of the time series"

If we integrate the expression PR would I be able to set the upper bound to something like the length of the target variable?